Docker desktop mac 3.0.2 release notes (#11931)
docker/docker.github.io: docker-for-mac/release-notes.md

> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
{: .important }

## Docker Desktop Community 3.0.2
2020-12-18

> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)

### Bug fixes and minor changes

- Avoid timeouts during `docker-compose up` by making cache invalidation faster. Fixes [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957).
- Avoid generating a spurious filesystem DELETE event while invalidating caches. Fixes [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124).

> - It is again possible to share directories in `~/Library` into a container except Desktop ones. Fixes [docker/for-mac#5115](https://github.com/docker/for-mac/issues/5115).

Reviewer: ```suggestion
- It is now possible to share directories in `~/Library` (except Docker Desktop data directories) with a container. Fixes [docker/for-mac#5115](https://github.com/docker/for-mac/issues/5115).
```

> - You will get a performance warning popup if you create a container that share home or user Library directory.

Reviewer: ```suggestion
- You will now see a performance warning pop-up message if you create a container that shares the `Home` or  a user `Library` directory.
```

### Known issues

- Some DNS addresses fail to resolve within containers based on Alpine Linux 3.13. See [docker/for-mac#5020](https://github.com/docker/for-mac/issues/5020).

> ## Docker Desktop Community 3.0.1

Reviewer: Could you update the 3.0.1 download link please?
